api/app.go

```diff
@@ -1233,7 +1233,7 @@ func isInternalEnv(envKey string) bool {
 }
 
 func internalEnvs() []string {
-	return []string{"TSURU_APPNAME", "TSURU_APP_TOKEN", "TSURU_SERVICE", "TSURU_APPDIR"}
```

Comment:
@morpheu I think that TSURU_SERVICE was never used, this is a kind of typo, right?